testdata/scripts/seed.txt

```diff
@@ -93,23 +96,33 @@ var teststringVar = "teststring"
 
 func main() {
 	if len(os.Args) > 1 && os.Args[1] == "funcName" {
-		println(originalFuncName())
+		println(OriginalFuncName())
+	} else if len(os.Args) > 1 && os.Args[1] == "importedFuncName" {
+		println(imported.OriginalFuncName())
```

Comment:
we should also include printing the package path, be it via runtime or reflect, as that's e.g. what `gob` uses too